ultraplot/tests/test_colorbar.py

```diff
@@ -290,6 +290,57 @@ def test_label_placement_colorbar():
         ax.colorbar(h, loc=loc, labelloc=labelloc)
 
 
+def test_label_rotation_colorbar():
+    """
+    Ensure that all potential combinations of colorbar
+    label rotation is possible.
+    """
+    cmap = uplt.colormaps.get_cmap("viridis")
+    mylabel = "My Label"
+    fig, ax = uplt.subplots()
+    cbar = ax.colorbar(cmap, labelloc="top", loc="right", vert=False, labelrotation=23)
+    # Get the label Text object
+    for which in "xy":
+        tmp = getattr(cbar.ax, f"{which}axis").label
+        if tmp.get_text() == mylabel:
+            label = tmp
```

Comment:
It looks like you didn't finish the test here.

Reply:
Thanks!